onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -178,6 +183,12 @@ void BasicBackend::Infer(Ort::CustomOpApi& ort, OrtKernelContext* context) {
   }
   // Get Output tensors
   LOGS_DEFAULT(INFO) << log_tag << "Inference successful";
+#ifndef NDEBUG
+  if (openvino_ep::backend_utils::IsDebugEnabled()) {
+  std::string& hw_target = (global_context_.device_id != "") ? global_context_.device_id : global_context_.device_type;
+  printPerformanceCounts(*infer_request_, std::cout, hw_target, true);
```

Comment:
why do we want to print perf counter info for all debug builds?

Reply:
> why do we want to print perf counter info for all debug builds?

The idea of bringing in this functionality was that the user will actually get to see what layer is run on what device for an Hetero Build. so, I incorporated this for debug mode. Any suggestions @smkarlap ?
Do we want to completely remove this?